src/linear_algebra/matrix.lean

```diff
@@ -936,3 +936,53 @@ def alg_equiv_matrix {R : Type v} {M : Type w} {n : Type*} [fintype n]
   [comm_ring R] [add_comm_group M] [module R M] [decidable_eq n] {b : n → M} (h : is_basis R b) :
   module.End R M ≃ₐ[R] matrix n n R :=
 h.equiv_fun.alg_conj.trans alg_equiv_matrix'
+
+/-- `std_basis` is the standard basis for vectors. -/
```

Comment:
Maybe the docstring would be clearer as: 
`std_basis_vec` is the standard basis of the free module R^n or something along those lines? 
Also, since you don't assume vector space, maybe not calling it vec would be better?

Reply:
I agree. I called it `vec` because of `matrix.mul_vec`, is there a better name for these things?

Reply:
standard basis vector seems reasonably common, even for the free module R^n. Otherwise I only know standard basis element.